swagger_server/controllers/default_controller.py

```diff
@@ -19,12 +18,13 @@ def distance_post(body):  # noqa: E501
     if connexion.request.is_json:
         sample = Sample.from_dict(connexion.request.get_json())  # noqa: E501
 
-        within_distance = get_db().query(DistanceORM).filter(
-            DistanceORM.s1 == sample.experiment_id and DistanceORM.d < 5
-        ).all()
+        neighbors = SampleNode.nodes.filter(name=sample.experiment_id)\
+            .neighbors.match(dist__lt=10)
 
         return ApiResponse(
             type='distance',
             sub_type='nearest-neighbor',
-            result={sample.s2: sample.d for sample in within_distance}
+            result=[{
+                'name': s.name
```

Comment:
Can we also return the distance as well. See the format specification in https://trello.com/c/wDicpcGt

Reply:
sure